src/api/.rubocop.yml

```diff
@@ -81,3 +81,311 @@ Rails/BulkChangeTable:
+    -  assert_project
+    -  assert_redirected_to
+    -  assert_response
+    -  assert_select
+    -  assert_xml_tag
+    -  author
+    -  authorize
+    -  basic_auth
+    -  belongs_to
+    -  boolean
+    -  build
+    -  call
+    -  can_create_attribute_in?
+    -  can_create_package_in?
+    -  can_modify?
+    -  category
+    -  change_column
+    -  change_column_default
+    -  change_column_null
+    -  check_access?
+    -  clean_older_than
+    -  click_button
+    -  click_link
+    -  column
+    -  command_name
+    -  comment!
+    -  commit
+    -  concat
+    -  connect
+    -  content
+    -  content_type
+    -  count
+    -  create
+    -  create!
+    -  create_missing_users_for
+    -  datetime
+    -  debug
+    -  default_server
+    -  define
+    -  delete
+    -  delete_element
+    -  delete_file
+    -  desc
+    -  describe
+    -  description
+    -  description!
+    -  destroy
+    -  disabled_for?
+    -  disturl
+    -  drop_table
+    -  email
+    -  enable_for_repository
+    -  end_with?
+    -  enqueue
+    -  entry
+    -  environment
+    -  error
+    -  escape_sql
+    -  exec_query
+    -  execute
+    -  exist?
+    -  exists_by_name
+    -  exists_by_project_and_name
+    -  exists_package?
+    -  exit
+    -  expect_with
+    -  extract_basic_user
+    -  extract_krb_user
+    -  factory_bot
+    -  fatal
+    -  feature
+    -  file_available?
+    -  fill_in
+    -  find
+    -  find_by
+    -  find_by_flag_and_status
+    -  find_by_id
+    -  find_by_login
+    -  find_by_login!
+    -  find_by_maintenance_db_project_id
+    -  find_by_name
+    -  find_by_name!
+    -  find_by_name_and_package
+    -  find_by_number
+    -  find_by_number!
+    -  find_by_project_and_name
+    -  find_by_repository_id
+    -  find_by_sql
+    -  find_by_title
+    -  find_in_batches
+    -  find_package
+    -  find_remote_project
+    -  first_or_create
+    -  float
+    -  force_hash
+    -  from
+    -  gem
+    -  generator
+    -  get
+    -  get_by_name
+    -  get_by_project_and_name
+    -  group
+    -  grouped
+    -  has_attribute?
+    -  has_element?
+    -  has_global_permission?
+    -  has_local_permission?
+    -  have_content
+    -  have_link
+    -  have_text
+    -  head
+    -  headers
+    -  hook_into
+    -  http_do
+    -  id
+    -  ignore_hosts
+    -  include
+    -  include?
+    -  index
+    -  info
+    -  instance_of?
+    -  instance_variable_get
+    -  instance_variable_set
+    -  integer
+    -  into
+    -  is_a?
+    -  is_in_group?
+    -  is_of_kind?
+    -  join
+    -  key?
+    -  kill
+    -  language
+    -  last_deployment
+    -  library
+    -  link
+    -  link_to
+    -  load
+    -  localize
+    -  local_role_check_with_ldap
+    -  login
+    -  login_king
+    -  maintainer
+    -  makeoriginolder
+    -  match_array
+    -  merge
+    -  merge_timeout
+    -  message
+    -  mkdir
+    -  must_be
+    -  name
+    -  new
```

Comment:
the ```new``` method shouldn't be ignored if we want to maintain consistency and avoid issues.

something like

```foo = MyClass.new 'bar' if bar.empty?``` is tricky

Reply:
removed new now - 34 offenses